src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/ScannedDocumentsHelper.java

```diff
@@ -73,25 +74,25 @@ public static void setExceptionRecordIdToScannedDocuments(
                         scannedDocument.url,
                         scannedDocument.deliveryDate,
                         exceptionRecord.id
-                    );
+                    ));
                 } else {
                     // do not update the document if it was received with some previous exception record
-                    return scannedDocument;
+                    return ImmutableMap.of("value", scannedDocument);
                 }
             })
             .collect(toList());
         caseData.put(SCANNED_DOCUMENTS, updatedScannedDocuments);
     }
 
     @SuppressWarnings("unchecked")
-    private static List<ScannedDocument> getScannedDocuments(Map<String, Object> caseData) {
+    public static List<ScannedDocument> getScannedDocuments(Map<String, Object> caseData) {
```

Comment:
It looks like this method has been made public just for the sake of calling it from tests. If this is the case - this should not be done.